aptible/environment.go

```diff
@@ -0,0 +1,73 @@
+package aptible
+
+import (
+	"github.com/aptible/go-deploy/client/operations"
+	"github.com/aptible/go-deploy/models"
+)
+
+type Environment struct {
+	Handle *string
+	ID     *int64
+	Type   *string
```

Comment:
see if this can be dropped or investigate as to why. This should just pull from the stack's type by default